src/main/java/uk/nhs/hee/tis/trainee/forms/service/LtftService.java

```diff
@@ -44,19 +45,22 @@
 @XRayEnabled
 public class LtftService {
 
+  private final AdminIdentity adminIdentity;
   private final TraineeIdentity traineeIdentity;
   private final LtftFormRepository ltftFormRepository;
   private final LtftMapper mapper;
 
   /**
    * Instantiate the LTFT form service.
    *
-   * @param traineeIdentity    The logged-in trainee.
+   * @param adminIdentity      The logged-in admin, for admin features.
+   * @param traineeIdentity    The logged-in trainee, for trainee features.
    * @param ltftFormRepository The LTFT repository.
    * @param mapper             The LTFT mapper.
    */
-  public LtftService(TraineeIdentity traineeIdentity, LtftFormRepository ltftFormRepository,
-      LtftMapper mapper) {
+  public LtftService(AdminIdentity adminIdentity, TraineeIdentity traineeIdentity,
```

Comment:
I'm not overly keen on having two different identity fields in the same service, it feels a bit like setting a trap for someone to fall in to later.

Once we add status change endpoints it will make more sense to split this service and create an abstract for the shared functionality like status updates.